google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -141,31 +141,10 @@ public static void addSupport(@NotNull AppEngineFlexibleFacet facet,
           );
 
           if (override == Messages.YES) {
-            defaultAppYaml.ifPresent(
-                appYaml -> WriteCommandAction.runWriteCommandAction(project,
-                    () -> {
-                      VirtualFile appYamlVirtualFile = contentRoots[0]
-                          .findFileByRelativePath("/src/main/appengine/app.yaml");
-                      if (appYamlVirtualFile != null) {
-                        PsiFile appYamlPsiFile =
-                            PsiManager.getInstance(project).findFile(appYamlVirtualFile);
-                        if (appYamlPsiFile != null) {
-                          Document appYamlDocument =
-                              PsiDocumentManager.getInstance(project)
-                                  .getDocument(appYamlPsiFile);
-                          if (appYamlDocument != null) {
-                            try {
-                              appYamlDocument.setText(
-                                  StringUtil.convertLineSeparators(
-                                      new String(Files.readAllBytes(appYaml),
-                                          Charset.defaultCharset())));
-                            } catch (IOException ioe) {
-                              logger.debug("Could not copy app.yaml text. " + ioe.getMessage());
-                            }
-                          }
-                        }
-                      }
-                    }));
+            defaultAppYaml.ifPresent(appYaml ->
```

Comment:
@etanshaul PTAL. You might be right that it improves readability of the addSupport() method.